src/main/haskell/kore/src/Kore/Unparser.hs

```diff
@@ -174,6 +174,9 @@ instance
             Domain.BuiltinMap _ -> Builtin.notImplementedInternal
             Domain.BuiltinList _ -> Builtin.notImplementedInternal
             Domain.BuiltinSet _ -> Builtin.notImplementedInternal
+            Domain.BuiltinInteger _ -> Builtin.notImplementedInternal
```

Comment:
I don't think we should implement unparsers for any of the internal representations because they do not have concrete Kore syntax. We should always be converting the pattern to the external representation before unparsing.

Reply:
While I agree with @ttuegel here, it would be nice to have a function to externalize and then unparse a pattern; this would allow a nicer output when generating an error, for example.